<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmployee</class>
 <widget class="QWidget" name="AddEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Express Delivery</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AddEmployee {
    background-color: #f5f5f5;
}

QPushButton#AddButton,
QPushButton#BackButton {
    font-size: 12px;
    background-color: transparent;
    color: #2c3e50;
    border: 1px solid #2c3e50; 
    border-radius: 8px;
    cursor: pointer;
    transition: background-color 0.3s;
}

QPushButton#AddButton:hover,
QPushButton#BackButton:hover {
    background-color: #2980b9;
}

QLabel {
    font-size: 10px;
    color: #2c3e50;
}

QLineEdit {
    background-color: transparent;
    color: #2c3e50;
    border: 1px solid #2c3e50; 
    border-radius: 8px;
}
</string>
  </property>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="NameLabel">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Nom:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameLineEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="ContactLabel">
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 10px;
}</string>
    </property>
    <property name="text">
     <string>Contact:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ContactLineEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="RoleLabel">
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 10px;
}</string>
    </property>
    <property name="text">
     <string>Role:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RoleLineEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="Hours_Label">
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 10px;
}</string>
    </property>
    <property name="text">
     <string>Volume Horaire:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HoursLineEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="PerformanceLabel">
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 10px;
}</string>
    </property>
    <property name="text">
     <string>Performance:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PerformanceLineEdit"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
